src/Faker/Provider/uz_UZ/PhoneNumber.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
Please remove this DocBlock, you'll be listed as contributor once your PR gets merged and every release each PR that got merged is listed as well.